rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -44,8 +44,25 @@ import com.normation.inventory.domain.NodeId
 
 
 /**
+ * The main class helping maps expected
+ * reports for a node
+ */
+case class RuleNodeExpectedReports(
```

Comment:
In fact what's bothering me is that there is no reference to a Node in that structure, and we say that it is related to a Node, meaning it is something external that defines it and we have no way to check it (and maybe that's good, but yet that makes me tickle)